app/components/ClientTaskList/ClientTaskList.tsx

```diff
@@ -14,21 +26,55 @@ import { TaskItem } from './TaskItem';
  * @returns {React.ReactElement} A JSX element representing the list of tasks.
  */
 const ClientTaskList = (): React.ReactElement => {
-  const { tasks, fetchTasks } = useTaskStore();
+  const { tasks, fetchTasks, reorderTask } = useTaskStore();
+
+  // https://docs.dndkit.com/api-documentation/sensors
+  const sensors = useSensors(
+    useSensor(PointerSensor, {
+      activationConstraint: {
+        distance: 8,
+      },
+    })
+  );
```

Comment:
This is implemented to make sure that the meatball menu is still available to the user - 

https://github.com/clauderic/dnd-kit/issues/591